deploy/001-source-timelocks.ts

```diff
@@ -12,66 +13,36 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
   const networkName = hre.network.name as SUPPORTED_NETWORKS;
   const live = hre.network.live;
 
-  const governorBravoDelegatorAddress = (await ethers.getContract("GovernorBravoDelegator")).address;
-
-  const getAdmin = () => {
-    return live ? governorBravoDelegatorAddress : deployer;
-  };
-
-  const delayConfig = {
-    hardhat: {
-      normal: 600,
-      fast: 300,
-      critical: 100,
-    },
-    bscmainnet: {
-      normal: 172800,
-      fast: 21600,
-      critical: 3600,
-    },
-    bsctestnet: {
-      normal: 600,
-      fast: 300,
-      critical: 100,
-    },
-    sepolia: {
-      normal: 600,
-      fast: 300,
-      critical: 100,
-    },
-    ethereum: {
-      normal: 172800,
-      fast: 21600,
-      critical: 3600,
-    },
+  const getAdmin = async () => {
+    return live ? (await ethers.getContract("GovernorBravoDelegator")).address : deployer;
   };
 
   await deploy("NormalTimelock", {
     contract: "Timelock",
```

Comment:
```suggestion
    contract: live ? "TimelockV8" : "TestTimelockV8",
```

Reply:
Currently this script fails because the delay is too small but using the Test/mock contract locally should solve the issue

Reply:
[c103b4d](https://github.com/VenusProtocol/governance-contracts/pull/57/commits/c103b4d7ddcbfba0046fb12a525a3d11f8326c71)